xlt-timerrecorder-chrome/src/page.js

```diff
@@ -204,3 +205,41 @@ function getPaintTimings(navigationStart) {
 
   return timings;
 }
+
+// --- Web Vitals --------------------------------------------------------------
+
+/**
+ * Store for Web Vital metrics recorded during the lifespan of a page.
+ * The entries are not the original metrics, but simplified ones with only the data we need.
+ */
+const webVitalsMetrics = [];
+
+/**
+ * Returns the entries in the metrics store. For now, no further processing needed.
+ */
+function getWebVitals() {
```

Comment:
Just out of curiosity: entries are always added to the array but never removed; is this correct? Or shouldn't we return a copy of the array and clear it before returning the copy to the caller?

Reply:
IMO we don't need to clear the store. It goes away when the new page is being loaded and will be recreated for that anew.

Reply:
I'm fine w/ that as long as there is no case where the same data is returned multiple times, esp. in context of SPAs that have only one page load.

Reply:
For sake of completeness: Resource timings are cleared upon retrieval whereas page load and paint timings may be returned multiple times. As WebVitals fall into the same category as page load and paint timings, we should be fine here.